kommunicate/src/main/java/io/kommunicate/zendesk/KmZendeskSendMessageTask.java

```diff
@@ -0,0 +1,61 @@
+package io.kommunicate.zendesk;
+
+import android.content.Context;
+import android.os.AsyncTask;
+
+import io.kommunicate.callbacks.KmCallback;
+import zendesk.chat.Attachment;
+
+public class KmZendeskSendMessageTask extends AsyncTask<Void, Void, KmZendeskApiModel> {
+    private Context context;
+    private String message;
+    private String displayName;
+    private String agentId;
+    private Integer conversationId;
+    private Long messageTimestamp;
+    //private boolean attachment;
```

Comment:
is this commented code required in future use?

Reply:
It was not required, removed it.